assets/img/icons/mfa/hotpFactor_70x70.svg

```diff
@@ -0,0 +1,6 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 140.00 140.00" preserveAspectRatio="xMidYMid meet">
+  <rect width="100%" height="100%" fill="#e3f3fe" />
+  <g transform="translate(0.000000,140.000000) scale(0.10,-0.10)" fill="#000000" stroke="none">
+    <path d="M640 1054 c-59 -37 -136 -64 -182 -64 l-28 0 0 -209 0 -209 28 -53 c23 -45 43 -64 132 -126 58 -40 110 -72 115 -73 13 0 172 105 208 139 16 14 39 44 50 66 21 37 22 54 22 250 l0 210 -45 6 c-55 8 -120 34 -181 71 -25 15 -50 28 -55 27 -5 -1 -34 -16 -64 -35z m123 -178 c45 -19 92 -70 106 -116 9 -29 17 -40 31 -40 27 0 25 -9 -7 -42 -33 -35 -40 -35 -75 -1 -33 32 -35 43 -8 43 26 0 25 19 -3 61 -29 42 -60 59 -115 59 -80 0 -135 -55 -135 -136 0 -99 108 -165 196 -119 22 11 34 12 49 4 26 -14 10 -33 -48 -55 -116 -44 -244 45 -244 170 0 133 133 223 253 172z m-55 -121 c2 -38 10 -57 30 -78 20 -21 23 -30 14 -39 -9 -9 -20 -4 -47 22 -32 31 -35 39 -35 93 0 50 2 58 18 55 13 -2 18 -15 20 -53z" fill="#0087d0"/>
+  </g>
+</svg>
```

Comment:
Purpose of this is to showcase what we expect from an SVG in order to render properly:

- SVG images must be spaced using `xMidYMid meet` to ensure they can render from within our beacon/round viewbox.
- Scaling needs to be considered by the uploader, since the image is shown in a few multiple views.

When this feature is added within our admin UI, let's make sure we have a preview available.

Reply:
If this is needed for clear rendering in SIW we should totally recommend it in the docs/admin UI.